ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py

```diff
@@ -206,3 +206,38 @@ def test_paddle_sqrt_(
         on_device=on_device,
         x=x[0],
     )
+
+
+# subtract_
+@handle_frontend_test(
+    fn_tree="paddle.tensor.math.subtract_",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```
This should always be `valid` - if only `float` data-type is supported then that's what `valid` will return.